policyengine_us/variables/gov/states/nm/tax/income/nm_subtractions.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class nm_subtractions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = (
+        "New Mexico income subtractions, excluding exemptions and deductions"
```

Comment:
Subtractions are always separate from exemptions and deductions
```suggestion
        "New Mexico income subtractions"
```